Wikipedia/Code/WMFArticlePreviewTableViewCell.m

```diff
@@ -30,11 +34,22 @@ @implementation WMFArticlePreviewTableViewCell
 
 #pragma mark - Setup
 
+- (void)blurAndShowLoadingIndicator {
+    self.blurView.hidden = NO;
```

Comment:
Should I maybe only add the blur and loading indicator views just before showing them and remove them after hiding them so we don't have extra views on every cell? Only used by random atm..